storage/io/src/main/java/org/openscience/cdk/io/PMPReader.java

```diff
@@ -240,11 +240,11 @@ private IChemFile readChemFile(IChemFile chemFile) {
                                 line = readLine();
                             }
                             if (chemObject instanceof IAtom) {
-                                atomids.put(Integer.valueOf(id), Integer.valueOf(modelStructure.getAtomCount()));
+                                atomids.put(id, modelStructure.getAtomCount());
                                 atomZOrders.put(Integer.valueOf((String) chemObject.getProperty(PMP_ZORDER)),
```

Comment:
There was some question elsewhere... I used the property in my CDK-using code supporting my PhD research. It's hard to figure out if others are using that property, but removing it would be a breaking change.

Reply:
I don't remember where the question was, but if it doesn't sound familiar, ignore this question.